test/classes/semver.js

```diff
@@ -5,52 +5,60 @@ const comparisons = require('../fixtures/comparisons.js')
 const equality = require('../fixtures/equality.js')
 const invalidVersions = require('../fixtures/invalid-versions')
 
-test('comparisons', t => {
+test('comparisons', (t) => {
   t.plan(comparisons.length)
-  comparisons.forEach(([v0, v1, opt]) => t.test(`${v0} ${v1}`, t => {
-    const s0 = new SemVer(v0, opt)
-    const s1 = new SemVer(v1, opt)
-    t.equal(s0.compare(s1), 1)
-    t.equal(s0.compare(v1), 1)
-    t.equal(s1.compare(s0), -1)
-    t.equal(s1.compare(v0), -1)
-    t.equal(s0.compare(v0), 0)
-    t.equal(s1.compare(v1), 0)
-    t.end()
-  }))
+  comparisons.forEach(([v0, v1, opt]) =>
```

Comment:
why the unrelated formatting changes?

Reply:
My ide automatically ran prettier I think, which fixed the line length for something eslint was complaining about, so I just kept the changes

I'll flip the rest of the lines back to what they were

Reply:
Yeh I did for the others but eslint couldn’t auto fix the indentation issue and I got lazy 😆